components/eventing-controller/controllers/subscription/nats/reconciler_test.go

```diff
@@ -150,253 +166,241 @@ func testCleanEventTypes(id int, eventTypePrefix, natsSubjectToPublish, eventTyp
 			ensureSubscriptionCreated(ctx, subscription)
 
 			Context("A Subscription without filters", func() {
```

Comment:
I am not, but my argument here is, that your example will produce the following output in case of a failing test.
```
"A Subscription without filters [By] should have no clean event types"
```
 so I think lower case reads nicer.